pages/en/lb4/Controllers.md

```diff
@@ -156,3 +156,52 @@ class HelloController {
 - A `Repository` is LoopBack's database abstraction. See [Repositories](Repositories.html) for more.
 - `@get('/messages')` creates the `Route` for the Operation using `app.route()`.
 - `@param.query.number` adds a `number` param with a source of `query`.
+
+## Handling Errors in Controllers
+
+In order to specify errors for controller methods to throw, the class `HttpErrors` is used. `HttpErrors` can be found in the `@loopback/rest` package.
+
+The example below shows how `HttpErrors` can be used inside a controller method and how it can be tested.
+
+```js
```

Comment:
I think most of the docs use `ts` not `js` ... switching to which would require you to add types for variables as per @jannyHou 's comment.

Reply:
@virkt25 yesterday I learned why we use `js` not `ts` lol... `.ts` doesn't recognize some keyword like `async`:

```ts
// a .ts file
async
```

```js
// a .js file
async
```

Reply:
:o interesting. I didn’t know that. Thanks for enlightening me @jannyHou